Description: get rid of loom
 due to loom depending on generator thus #1003070
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,7 +47,3 @@
 
 [features]
 tls = []
-
-[target."cfg(loom)".dependencies.loom]
-version = "0.5"
-features = ["checkpoint"]
